widget/select.go

```diff
@@ -145,7 +195,7 @@ func (s *Select) Resize(size fyne.Size) {
 
 func (s *Select) optionTapped(text string) {
 	s.SetSelected(text)
-	s.popUp = nil
```

Comment:
code elsewhere checks for nil of c.popUp, is this safe?

Reply:
Not sure. But I changed optionTapped() to be identical to the one on develop, and it is ok.